.github/workflows/pr-title.yml

```diff
@@ -0,0 +1,30 @@
+name: Lint pull request title
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+
+permissions:
+  contents: read
+
+jobs:
+  main:
+    permissions:
+      pull-requests: read
+      statuses: write
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@0723387faaf9b38adef4775cd42cfd5155ed6017 # v5.5.3
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          scopes: |
+            foreman
+            deps
```

Comment:
I wonder if we should continue to have `deps`. Thinking about it, the only time this would occur is with Renovate updates to CI workflow dependencies. But in that case, `repo` seems fitting as well.

Reply:
I think that should be another PR.

Reply:
I think we should remove it here, and have another PR to re-configure Renovate.